Doc updates (#10)
kafka4beam/wolff: changelog.md

* 1.5.4

>   - Fix: no delay before the first re-connect attmpt after disconnected while idling.

Reviewer: ```suggestion
  - Fix: no delay before the first re-connect attempt after disconnected while idling.
```

* 1.5.3

>   - Enhancement: refine logging, reduce the number of retry logs, report error level log after each 10 times of failure

Reviewer: ```suggestion
  - Enhancement: refine logging, reduce the number of retry logs, report error level log every 10 failures
```
